test/FunctionalTests/Microsoft.OData.Core.Tests/Build.NetFramework/Microsoft.OData.Core.Tests.csproj

```diff
@@ -454,6 +454,7 @@
     <Compile Include="..\WriterUtilsTests.cs" />
     <Compile Include="..\ScenarioTests\Streaming\ODataJsonLightStreamReadingTests.cs" />
     <Compile Include="..\ScenarioTests\Streaming\ODataJsonLightStreamWritingTests.cs" />
+    <Compile Include="SelectExpandClauseExtensionsTests.cs" />

```

Comment:
meanwhile, please add the cs file into other two project. One is the net 3.5, and the other is .net standard. Please check other cs files